polar_route/mesh_generation/aggregated_cellbox.py

```diff
@@ -149,3 +149,6 @@ def contains_point(self, lat, long):
             if (long >= self.boundary.get_long_min()) & (long <= self.boundary.get_long_max()):
                 return True
         return False
+    
+    def get_bounds (self):  # adding this method to be compatabile with the Cellbox and use the NeighbourGraph.get_neighbour_case method
```

Comment:
Does this method need to be duplicated? Could we just rename the original method to be consistent with cellbox.py? Also I'm not sure I understand the reference to `NeighbourGraph.get_neighbour_case` here.

Reply:
I was trying to avoid having multiple files changes to track things. But you are correct. I have renamed it.